ledger/metrics/src/main/scala/com/daml/metrics/HttpJsonApiMetrics.scala

```diff
@@ -25,43 +30,76 @@ class HttpJsonApiMetrics(override val prefix: MetricName, override val registry:
+    DropWizardMetrics.timer(prefix :+ "incoming_json_parsing_and_validation_timing")
   // Meters how long the construction of the response json payload takes
-  val responseCreationTimer: Timer = timer(prefix :+ "response_creation_timing")
+  val responseCreationTimer: Timer = DropWizardMetrics.timer(prefix :+ "response_creation_timing")
   // Meters how long a find by contract key database operation takes
-  val dbFindByContractKey: Timer = timer(prefix :+ "db_find_by_contract_key_timing")
+  val dbFindByContractKey: Timer =
+    DropWizardMetrics.timer(prefix :+ "db_find_by_contract_key_timing")
   // Meters how long a find by contract id database operation takes
-  val dbFindByContractId: Timer = timer(prefix :+ "db_find_by_contract_id_timing")
+  val dbFindByContractId: Timer = DropWizardMetrics.timer(prefix :+ "db_find_by_contract_id_timing")
   // Meters how long processing of the command submission request takes on the ledger
   val commandSubmissionLedgerTimer: Timer =
-    timer(prefix :+ "command_submission_ledger_timing")
+    DropWizardMetrics.timer(prefix :+ "command_submission_ledger_timing")
   // Meters http requests throughput
-  val httpRequestThroughput: Meter = meter(prefix :+ "http_request_throughput")
+  val httpRequestThroughput: Meter = DropWizardMetrics.meter(prefix :+ "http_request_throughput")
   // Meters how many websocket connections are currently active
-  val websocketRequestCounter: Counter = counter(prefix :+ "websocket_request_count")
+  val websocketRequestCounter: Counter =
+    DropWizardMetrics.counter(prefix :+ "websocket_request_count")
   // Meters command submissions throughput
   val commandSubmissionThroughput: Meter =
-    meter(prefix :+ "command_submission_throughput")
+    DropWizardMetrics.meter(prefix :+ "command_submission_throughput")
   // Meters package uploads throughput
-  val uploadPackagesThroughput: Meter = meter(prefix :+ "upload_packages_throughput")
+  val uploadPackagesThroughput: Meter =
+    DropWizardMetrics.meter(prefix :+ "upload_packages_throughput")
   // Meters party allocation throughput
-  val allocatePartyThroughput: Meter = meter(prefix :+ "allocation_party_throughput")
+  val allocatePartyThroughput: Meter =
+    DropWizardMetrics.meter(prefix :+ "allocation_party_throughput")
+
+  object OpenTelemetryMetrics extends OpenTelemetryFactory {
+    override val prefix: MetricName = HttpJsonApiMetrics.this.prefix
+    override val otelMeter: OtelMeter = HttpJsonApiMetrics.this.otelMeter
+  }
+
+  // golden signals
+  // http metrics
+  val httpRequestsTotal: Counter = OpenTelemetryMetrics.counter(prefix :+ "requests_total")
+  val httpErrorsTotal: Counter = OpenTelemetryMetrics.counter(prefix :+ "errors_total")
+  val httpLatency: Timer = OpenTelemetryMetrics.timer(prefix :+ "requests_duration_seconds")
+  val httpRequestsBytesTotal: Counter =
+    OpenTelemetryMetrics.counter(prefix :+ "requests_bytes_total")
```

Comment:
Nit: I find the names for request and response sizes slightly confusing here, i.e., it's not apparent what exactly they measure. I'd suggest further clarifying them like `request/response_size_bytes_total` or alternatively `request/response_payload_size_bytes_total` instead.

Reply:
I tried to see if there was some 'best practices' about naming these metrics, but I didn't find much.
In the few examples I found, the names seems to be limited to `requests_bytes` and `responses_bytes`.
But I am not opposed to using something a bit more expressif.

Reply:
So would you be OK with suffixing these metrics with `payload` (or `size` alternatively), like `request_payload_bytes_total` and `response_payload_bytes_total`? I think that would improve expressiveness.
Otherwise I'm OK with keeping them as-is, i.e., we can easily change this afterwards.